rclcpp/src/rclcpp/memory_strategies.cpp

```diff
@@ -14,6 +14,8 @@
 
 #include "rclcpp/memory_strategies.hpp"
 
+#include <memory>
```

Comment:
The same question as https://github.com/ros2/demos/pull/80#discussion_r81456795 applies here.